src/recipe.rst

```diff
@@ -42,11 +47,12 @@ Step-by-step Instructions
    example/meta.yaml>`_
    file from the example directory. Please leave the example directory
    unchanged!
-4. Edit the copied recipe (meta.yml) as needed.
-5. Generate the SHA256 key as described in the example recipe using the
-   ``openssl`` tool. As an alternative you can also go to the package
-   description on `PyPi <https://pypi.org>`_ from which you can directly copy
-   the SHA256.
+4. Edit the copied recipe (meta.yml) as needed. For details, see 
+   `Writing the ``meta.yml`` <https://conda-forge.org/docs/meta.html>`
```

Comment:
I think these inner backticks may throw off the syntax.  Try quotes instead?

Reply:
Good call, fixing that.